exporter/opentelemetry-exporter-otlp-proto-common/src/opentelemetry/exporter/otlp/proto/common/_internal/metrics_encoder/__init__.py

```diff
@@ -65,9 +65,21 @@ class OTLPMetricExporterMixin:
     def _common_configuration(
         self,
         preferred_temporality: Dict[type, AggregationTemporality] = None,
+        preferred_aggregation: Dict[type, "opentelemetry.sdk.metrics.view.Aggregation"] = None,
+        preferred_aggregation: Dict[
+            type, "opentelemetry.sdk.metrics.view.Aggregation"
+        ] = None,
```

Comment:
Thank you for putting in this issue and PR!

One of these `preferred_aggregation` should be removed, else there will be a `SyntaxError: duplicate argument 'preferred_aggregation' in function definition`.

Reply:
Thank you! Fixed on 00ef835757791d44c07bd4fc172d080609247060